spec/requests/api/braintree_spec.rb

```diff
@@ -1067,48 +1080,53 @@
         end
       end
 
-      let!(:member) { create :member, email: user_params[:email], postal: nil }
-      let!(:customer) {
-        create :payment_braintree_customer, :with_payment_methods,
-               member: member,
-               customer_id: 'test',
-               card_last_4: '4843',
-               payment_methods: 3
-      }
+      let!(:member)   { create :member, email: user_params[:email], postal: nil }
+      let!(:customer) { create(:payment_braintree_customer, member: member, customer_id: 'test', card_last_4: '4843') }
+
+      before do
+        3.times do
+          Payment::BraintreePaymentMethod.create(customer: customer)
```

Comment:
might be nice to use a factory here `create :payment_braintree_payment_method, customer: customer`